jenkins-integration/beaker/install/foss/40_add_puppet_bin_to_path.rb

```diff
@@ -0,0 +1,5 @@
+test_name "Add puppet binaries to the $PATH"
+
+# hosts.each do |host|
+  on(hosts, 'echo "export PATH=\$PATH:/opt/puppetlabs/bin" >> ~/.bashrc')
+# end
```

Comment:
We should get rid of this file if it's only meant for us humans if/when we log in to the system :smile: